docs/src/modules/javascript/pages/views.adoc

```diff
@@ -38,88 +50,57 @@ include::example$js-customer-registry/customer_view.proto[tag=unary]
 <1> Without `stream` when expecting single result.
 
 When no result is found, the request fails with gRPC status code `NOT_FOUND`. A streamed call completes with an empty stream when no result is found.
+====
 
-[#query]
-== Query
-
-Define View queries in a language that is similar to SQL. The following examples illustrate the syntax.
-
-All customers without any filtering conditions (no WHERE clause):
-[source,proto,indent=0]
-----
-SELECT * FROM customers
-----
+See <<#query>> for examples of valid query syntax.
 
-Customers with a name matching the `customer_name` property of the request message:
-[source,proto,indent=0]
-----
-SELECT * FROM customers WHERE name = :customer_name
-----
-
-Customers with matching `customer_name` AND `city` properties of the request message:
-[source,proto,indent=0]
-----
-SELECT * FROM customers WHERE name = :customer_name AND address.city = :city
-----
-
-Customers in city matching a literal value:
-[source,proto,indent=0]
-----
-SELECT * FROM customers WHERE address.city = 'New York'
-----
-
-Possible filter predicates are:
-
-* `=` equals
-* `!=` not equals
-* `>` greater than
-* `>=` greater than or equals
-* `<` less than
-* `\<=` less than or equals
+[#register-view]
+=== Register the View
 
-The filter conditions can be combined with `AND`/`OR`.
+In the View implementation, register the View with Akka Serverless. In addition to passing the service descriptor and a unique identifier, pass any descriptors that define state. In this example, the `customer_domain.proto` descriptor defines the Value Entity state:
 
-[source,proto,indent=0]
+.customer-value-entity-view.js
+[source,js,indent=0]
 ----
-SELECT * FROM customers WHERE
-  name = :customer_name AND address.city = 'New York' OR
-  name = :customer_name AND address.city = 'San Francisco'
+include::example$js-customer-registry/customer-value-entity-view.js[tag=register]
 ----
 
-[#register-view]
-== Registering the View
-
-Once you've defined a View, register it with the `AkkaServerless` server by invoking the `addComponent` function.
+To use the View in the client, invoke the `addComponent` function. For example:
```

Comment:
I don't think this should be `in the client`. It's registering the view as a component in the service / user function (or whatever we're using for this now). `client` is confusing in this context.

Reply:
The original says:

> Once you've defined a View, register it with the `AkkaServerless` server by invoking the `addComponent` function.

which makes it clear that this is registering in the server or service.

Reply:
hmm, the code is in the client, no, index.js? And, I received a comment that talking about a serverless server was a bit confusing. Changed to: Invoke the `addComponent` function to register the view with the service.

Reply:
Not sure why `index.js` implies a client. Using `service` is good, but has also been seen as ambiguous. Either way, the component is registered via the SDK with the thing that gets deployed as an akka serverless service. A `client` should be reserved for clients that call the deployed service.

Reply:
See https://github.com/lightbend/akkaserverless-javascript-sdk/pull/139/files